doc/tech/glossary.md

```diff
@@ -20,15 +20,15 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Network Function Virtualisation (NFV)**: The concept of separating network functions from the hardware they run on by using a virtual hardware abstraction layer.
 - **Network Function Virtualisation Infrastructure (NFVI)**: The totality of all hardware and software components used to build the environment in which a set of VAs are deployed (also referred to as cloud infrastructure).
- >_*Note:*_ The NFV-Infrastructure can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the cloud infrastructure. cloud infrastructure and VNF are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
-
+ >_*Note:*_ The NFVI can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the cloud infrastructure. **NFVI** and **VNF** are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
+- **Cloud Infrastructure**: A generic term covering **NFVI** and **CaaS** capabilities - essentially the infrastructure on which either a VNF or CNF can be executed.
```

Comment:
We should not use "either .. or" here. A cloud infrastructure should be able to host VM based and containerized workloads together. Also saying "either a VNF or a CNF" is weird since a CNF can be a VNF and a VNF can be a CNF.
Can we say "... on which VNFs and/or CNFs can be executed"? Another option would be to again use "workloads".

Reply:
Yes, I prefer workload - updated in https://github.com/cntt-n/CNTT/pull/1328/commits/7b0dc9ec9b4703f0ed570a8312f9e02dd73d8af5